dist/common/string.d.ts

```diff
@@ -16,6 +16,21 @@
  * ```
  */
 export declare function createSearch<TObj>(searchText: string, stringifier?: (obj: TObj) => string): (obj: TObj) => boolean;
+export declare const VOWELS: string[];
+/**
+ * Pluralizes a word based on the number given.
+ * Handles -es and -ies.
+ *
+ * @param override - A custom string to be appended instead for plurals. Useful for words that don't follow the standard rules.
+ *
+ * @example
+ * ```tsx
+ * pluralize('Dog', 1) // Dog
+ * pluralize('Dog', 2) // Dogs
+ * pluralize('Monarch', 2, "s") // Monarchs
+ * ```
+ */
+export declare function pluralize(str: string, n: number, override?: string): string;
```

Comment:
unneeded for this pr, I think

Reply:
Yeah we should probably figure out how to manage these nicely

Reply:
I still can revert building

Reply:
Huh, seems like it just not builded commit
![image](https://github.com/user-attachments/assets/5298ae6b-fdca-4e94-9140-b3ade72a3b5a)

Life without Juke Build is like that...
Just like Paradise :D

Reply:
It's because I didn't build it for my PR, and because this PR builds it's included here

Reply:
> It's because I didn't build it for my PR, and because this PR builds it's included here

Soo... i need to build it or not?